scripts/globals/mobskills/target_analysis.lua

```diff
@@ -0,0 +1,65 @@
+---------------------------------------------
+-- Target Analysis
+--
+-- Description: AoE Absorb All with randomness
+-- Type: Magical
+---------------------------------------------
+require("scripts/globals/mobskills")
+require("scripts/globals/status")
+require("scripts/settings/main")
+local ID = require("scripts/zones/Sealions_Den/IDs")
+local ID2 = require("scripts/zones/Apollyon/IDs")
+---------------------------------------------
+local mobskill_object = {}
+
+local attributes =
```

Comment:
This table name should probably be more specific.  Its not a table of attributes, but attribute down effects.